src/components/Button.scss

```diff
@@ -1,15 +1,58 @@
-.Button {
+.general_button {
   width: 30%;
+  font-family: Montserrat;
+  outline:none;
+  display:block;
+  border:none;
+  text-align: center;
+}
+
+.square {
   border-radius: 4px;
+  font-size: 16px;
+  height: 40px;
+}
+
+.square_button {
   background-color: #09628D;
   color: white;
-  text-align: center;
-  height: 40px;
   line-height: 40px;
-  border:none;
+  box-shadow: 0 2px 4px 0 #A3C0CC;
 }
 
-.Button:hover {
+.square_button:hover {
   background-color: #09769F;
   outline: none;
 }
+
+.blueborder_button {
+  border: 2px solid #09719B;
+  background-color: #FFFFFF;
+  box-shadow: 0 1px 1px 0 #D9DBDC;
+  font-weight: 500;
+  color: #09719B;
+}
+
+.round {
+  border-radius: 20px;
+  color: #FFFFFF;
+  height: 40px;
+  line-height: 40px;
+}
+
+//TODO : Check with UX Designer about linear gradient
```

Comment:
Yes when we discussed earlier there still work to be done on the linear gradients im waiting for that, thats why i added it there.